playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_repo.yml

```diff
@@ -0,0 +1,34 @@
+- name: set latest tag from git tag
```

Comment:
This doesn't look like it's building the openshift-ansible repo. Maybe a more semantic name?